source/2019-12-02-countdown-to-the-new-year-ember-modifier.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Countdown to The New Year- Ember Modifier
+author: Melanie Sumner
+tags: 2019, Addons, DecEmber
+responsive: true
+---
+
+## Day 3
+
+This is the third in our DecEmber series– _"Countdown to The New Year: 31 Days of Ember Addons"_. We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!
+
+Today we are going to look at a rising star in our addon ecosystem- [ember-modifier](https://emberobserver.com/addons/ember-modifier). 
+
+READMORE
+
+### What It Does
+
+The `ember-modifier` addon provides an API for authoring element modifiers in Ember. It mirrors Ember's [helper API](https://api.emberjs.com/ember/3.14/classes/Ember.Templates.helpers), and has forms for writing both simple functional modifiers and more complicated class modifiers.
+
+A modifier can add special, automatic behavior to a DOM element, such as tracking scroll position or adding new event listeners. After you write a modifier, you can use it in a template like this:
```

Comment:
This one should fix the linter error on CI right now:

```
source/2019-12-02-countdown-to-the-new-year-ember-modifier.md
  20:20-20:27  warning  `special` may be insensitive, use `has a disability`, `person with a disability`, `people with disabilities` instead  special  retext-equality
```

This suggestion should fix it I think:

```suggestion
<!--alex ignore special-->
A modifier can add special, automatic behavior to a DOM element, such as tracking scroll position or adding new event listeners. After you write a modifier, you can use it in a template like this:
```

Reply:
Changed the world to "customized" - what do you think?

Reply:
Yes, that sounds awesome!